src/components/editor/AuthorListController.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController'
+
+export default class AuthorListController extends FalcorController {
+  static getFalcorPath(params) {
+    return [];
+  }
+
+  render() {
+    return <div><p>Hi</p></div>;
+  }
+}
```

Comment:
yeah, all text files should end in a newline char or they are not considered a complete line.